timespan/update.go

```diff
@@ -12,17 +12,17 @@ import (
 
 // UpdateTimeSpan update a time span
```

Comment:
For me your solution kinda works with the one remark in the other comment.
![set-zero.webm](https://github.com/user-attachments/assets/cfaa7915-166a-4f3c-b199-3ef43e67b323). This MR changes many lines that are not necessary, this should be enough: https://github.com/traggo/server/compare/set-to-zero?expand=1

Reply:
Huh very weird. I tried your solution, but it doesn´t work for me. It still sends the whole number down. Let me try and just use your PR that you made and see if that fixes this. 

But if you arleady have a working solution that works then we can merge that, I guess my machine is just weird ;/. 

Sorry for the inconvenience

Reply:
How are you starting the UI? Which port do you use?